### ISSUE TITLE
webhook event

### ISSUE THREAD

**Issue by Prodent33** [devinweb/laravel-youcan-pay#5]
The project is currently in testing and has been integrated with your payment platform using  "DevinWeb" package. Everything is working well in test mode, where payments are received using the package's test bank card, and transactions appear in the control panel and transaction page in youcanpay dashboard as a table format that contains payment information and customer information.

Everything is working well in this part, but when connecting the platform to the project's database through the "webhook" to send the event responsible for updating the payment status from "pending" to "paid," a problem has emerged recently for almost five days . We are not receiving any events through the webhook despite the successful payment and the information appearing on the transaction page. We have confirmed that the webhook has been linked to youcanpay platform and our project by adding the project link and integrating it with the webhook and selecting the type of event we want to receive.

Therefore, we kindly ask for your assistance in verifying the issue and determining whether other people are also facing this problem or if the issue is on our end. A significant portion of the project depends on the message sent through your webhook. Thank you.

The webhook was linked by adding the website link with the webhook link in the following format: https://5966-196-121-1-105.ngrok-free.app/youcanpay/webhook

**Comment by darbaoui** [comment]
@Prodent33 The issue still exists from your side, in the webhook you need to check the status from youcanpay dashboard if its fired or not, the webhook in this package works if there's an event received from youcanpay

**Comment by Prodent33** [comment]
@darbaoui  : Thank you for your response to the message.

Yes the problem still exists , the YoucanPay platform not sending the webhooks event for updating the status of the  transaction to "paid" when the agreed amount is deducted from the customer . It should be noted that the transaction status appears on the youcanPay dashboard as "Paid"  . The only thing missing is receiving the Event from the webhook .Therefore, I want to inquire about the problem, whether it affects all clients or if the problem is in the way the webhooks are connected.

**Comment by Prodent33** [comment]
> 
Thank you for your response to the message.

Yes the problem still exists , the YoucanPay platform not sending the webhooks event for updating the status of the transaction to "paid" when the agreed amount is deducted from the customer . It should be noted that the transaction information appears on the youcanPay dashboard as "Paid" . The only thing missing is receiving the notification from the webhook . Therefore, I want to inquire about the problem, whether it affects all clients or if the problem is in the way the webhooks are connected

**Comment by Prodent33** [comment]
video show webhook event issue.mov
<https://drive.google.com/file/d/1b4cZavc6488W-NOlWYd4LIscqtbIHyGZ/view?usp=drive_web>


هذا الفيديو يوضح مشكلة عدم إرسال منصة يوكان باي للويب هوك إفنت الخاص بتحديث
حالة التحويلة المالية إلى مدفوع عند الاقتطاع من العميل المبلغ المتفق عليه .
مع العلم أنه في صفحة التحويلات في منصة يوكان تظهر معلومات التحويلة وبأنه تم
اقتطاع المبلغ من العميل ينقص فقط تلقي الإشعار من الويب هوك . لذلك أريد
الاستعلام عن المشكل  هل يعم جميع زبنائكم أم المشكل في طريقة ربط الويب هوك

This video illustrates the problem of the YoucanPay platform not sending
the webhooks event for updating the status of the transaction to "paid"
when the agreed amount is deducted from the customer. It should be noted
that the transaction information appears on the youcanPay dashboard . The
only thing missing is receiving the notification from the webhook .
Therefore, I want to inquire about the problem, whether it affects all your
clients or if the problem is in the way the webhooks are connected.


Le mar. 11 avr. 2023 à 19:38, darbaoui imad ***@***.***> a
écrit :

**Comment by Prodent33** [comment]
video show webhook event issue.mov
<https://drive.google.com/file/d/12ZjZosW4XePA2nRsXEJ30kSWfG7oBe_J/view?usp=drive_web>
هذا الفيديو يوضح مشكلة عدم إرسال منصة يوكان باي للويب هوك إفنت الخاص بتحديث
حالة التحويلة المالية إلى مدفوع عند الاقتطاع من العميل المبلغ المتفق عليه .
مع العلم أنه في صفحة التحويلات في منصة يوكان تظهر معلومات التحويلة وبأنه تم
اقتطاع المبلغ من العميل ينقص فقط تلقي الإشعار من الويب هوك . لذلك أريد
الاستعلام عن المشكل  هل يعم جميع زبنائكم أم المشكل في طريقة ربط الويب هوك

This video illustrates the problem of the YoucanPay platform not sending
the webhooks event for updating the status of the transaction to "paid"
when the agreed amount is deducted from the customer. It should be noted
that the transaction information appears on the youcanPay dashboard . The
only thing missing is receiving the notification from the webhook .
Therefore, I want to inquire about the problem, whether it affects all your
clients or if the problem is in the way the webhooks are connected.

Le mar. 11 avr. 2023 à 19:38, darbaoui imad ***@***.***> a
écrit :

**Comment by darbaoui** [comment]
Please give me the access to the video, but if the webhook events are not fired from youcanpay you should contact them to give you an explanation why not fired.

**Comment by Prodent33** [comment]
"Please confirm for me if you have watched the video. I will be waiting for
your answer."


Le jeu. 13 avr. 2023 à 14:49, darbaoui imad ***@***.***> a
écrit :

**Comment by darbaoui** [comment]
I watched the video,there's no information abou the webhook if its fired or not, so you need to contact youcanpay and ask them why the webhook doesnt fired?

**Comment by darbaoui** [comment]
https://youtu.be/XJBZkLErIMQ

**Comment by Prodent33** [comment]
شكرا لك أخي صراحة معرفتش واش المشكل عند كلشي او فقط عندي ، فاش كنهضر معاهم
هوما معارفينش كاع علاش كنهضر . كنضن انت فهمتي المشكل اللي عندي


Le sam. 15 avr. 2023 à 01:23, darbaoui imad ***@***.***> a
écrit :

**Comment by darbaoui** [comment]
I need to help you please in youcanpay in the transaction page please send us a screenshot of all the page include the webhook section

**Comment by Prodent33** [comment]
"I'm sorry for the inconvenience . today , When I tried again, it seems
that the web hook is working properly without any problems. i dont know how
and I didn't update anything in my code, but I had contacted youcan support
to look into the problem. Now, I don't know if things will remain stable in
this way or if the problem will reappear again!"


Le dim. 16 avr. 2023 à 22:29, darbaoui imad ***@***.***> a
écrit :
